docs/devenv_setup.md

```diff
@@ -32,6 +32,7 @@ virt-install \
     --os-type generic \
     --events on_reboot=restart \
     --cdrom ./${ISONAME} \
+    --wait \
```

Comment:
Would it be ok to add a time value (in minutes) to this ? I do not know what the default is if you do not include a time value. I have been using --wait 5, and it works ok. Thoughts ?

Reply:
I responded below.